Mathlib/Init/Logic.lean

```diff
@@ -300,11 +300,32 @@ end Decidable
 #align decidable_of_decidable_of_eq decidable_of_decidable_of_eq
 #align or.by_cases Or.by_cases
 
-alias instDecidableOr ← Or.decidable
-alias instDecidableAnd ← And.decidable
-alias instDecidableNot ← Not.decidable
-alias instDecidableIff ← Iff.decidable
+/- Disjunctions are decidable. We're not using alias because this doesn't currently produce
+decidable definitions. -/
+def Or.decidable := @instDecidableOr
```

Comment:
computable aliases have been implemented recently. After merging master, these definitions can be changed (back) to aliases (I can also do this next week, if you prefer).

Reply:
This is from the merge with `Logic.lean` not `Ordnode.lean`

Reply:
No, these are things I changed to make `Ordnode` work. I changed them back